src/zope/publisher/paste.py

```diff
@@ -64,8 +64,10 @@ def request(self, environ):
 
 def get_egg(name, group):
     if '#' in name:
-        egg, entry_point = name.split('#', 1)
+        _, name = name.split('#', 1)
     else:
-        egg, entry_point = name, 'default'
+        _, name = name, 'default'
```

Comment:
```suggestion
        name = 'default'
```

No need to assign to an unused variable.